pages/common/duplicity.md

```diff
@@ -0,0 +1,23 @@
+# duplicity
+
+> Creates incremental, compressed, encrypted, and versioned backups and optionally uploads them to a variety of backend services.
+
+- Backup a folder via ftps to a remote machine, encrypting with a password:
+
+`FTP_PASSWORD={{ftp_login_password}} PASSPHRASE={{encryption_password}} duplicity {{path/to/source/directory}} {{ftps://user@remotehost.tld/target/directory/path/}}`
```

Comment:
> While I do agree they clutter the example, unfortunately they are required 

Hmm I see. Weird why they would make it an env var instead of a flag.